apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -2422,20 +2519,20 @@ export class CaseNotificationService extends BaseNotificationService {
     })
 
     promises.push(
-      this.sendEmail(
+      this.sendEmail({
         subject,
         html,
-        theCase.prosecutor?.name,
-        theCase.prosecutor?.email,
-      ),
-      this.sendEmail(
+        recipientName: theCase.prosecutor?.name,
+        recipientEmail: theCase.prosecutor?.email,
+      }),
+      this.sendEmail({
         subject,
         html,
-        theCase.defenderName,
-        theCase.defenderEmail,
-        undefined,
-        !theCase.defenderNationalId,
-      ),
+        recipientName: theCase.defenderName,
+        recipientEmail: theCase.defenderEmail,
+        attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.